# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1507234370 14400
#      Thu Oct 05 16:12:50 2017 -0400
# Node ID 9182a640afbef0f486bee67026c765d93b055d10
# Parent  16c8ec7f08673405feaabd503f23809570ef15ca
* acinclude.m4: Also search for qt4-specific qscintilla lib names.

diff --git a/m4/acinclude.m4 b/m4/acinclude.m4
--- a/m4/acinclude.m4
+++ b/m4/acinclude.m4
@@ -1166,17 +1166,17 @@ dnl
 AC_DEFUN([OCTAVE_CHECK_QSCINTILLA], [
   qt_version="$1";
   use_qscintilla=no
   warn_qscintilla=""
 
   ## Check for Qt libraries
   case "$qt_version" in
     4)
-      octave_qscintilla_libnames=qscintilla2
+      octave_qscintilla_libnames="qscintilla2 qscintilla2-qt4 qscintilla2_qt4 qt4scintilla2"
     ;;
     5)
       octave_qscintilla_libnames="qscintilla2-qt5 qscintilla2_qt5 qt5scintilla2"
     ;;
     *)
       AC_MSG_ERROR([Unrecognized Qt version $qt_version])
     ;;
   esac
